### ISSUE TITLE
[Feature Request] Port Slick Greeter's Chameleonic Colour into a Colour Tag for Theme-makers

### ISSUE THREAD

**Issue by feren** [linuxmint/Cinnamon#6605]
Basically, Slick Greeter currently still has the Chameleonic Colour Ability back from its life as Unity Greeter, which can be seen from the Power Dialog's colour on the screen depending on what wallpaper it is. Now what'd be really cool is if Cinnamon had that feature, and you could get that by using a colour tag in the Cinnamon theme's CSS probably called 'chameleon' to go with the other ones such as 'transparent' and '#HEXCODEHERE', and when Cinnamon sees that item, to then colour that section in a similar fashion to Slick Greeter's Shutdown Dialog...

Basically, ability for theme-maker to have bits of their Cinnamon theme coloured by the colour mostly present on the wallpaper. Oh, and transparency for that colour on top of that would also be nice.

**Comment by JosephMcc** [comment]
Many thanks for contributing to Cinnamon. Your suggestion was reviewed.

For more information on our workflow and feature requests, read https://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/faq.html.
